alfasim_sdk/variables.py

```diff
@@ -0,0 +1,47 @@
+import numbers
+from enum import Enum
+from typing import Optional
+
+import attr
+from attr import attrib
+from attr.validators import instance_of, optional
+from barril.units import UnitDatabase
+
+from alfasim_sdk._validators import check_for_valid_unit, check_string_is_not_empty
+
+
+class Visibility(Enum):
+    Internal = 'internal'
+    Output = 'output'
+
+
+class Location(Enum):
+    Center = 'center'
+    Face = 'face'
+
+
+class Scope(Enum):
+    Energy = 'energy'
+    Field = 'field'
+    Global = 'global'
+    Layer = 'layer'
+    Phase = 'phase'
+    # TODO ASIM - 2348 Add Scope.ContinuousField and Scope.DispersedField
+    # ContinuousField = 'continuous_field'
+    # DispersedField = 'dispersed_field'
+
+
+@attr.s(kw_only=True)
+class Variable():
```

Comment:
`Variable` is such a generic name. Perhaps we should call it at least `SecondaryVariable`, since in the future we may be able to also add `PrimaryVariable`s (I don't know yet how this would work, though). What do you think?

Reply:
Sounds good.